contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -435,6 +440,44 @@
       clearSelections() {
         this.selected = [];
       },
+
+      sortNodes(selected) {
+        const selectedNodes = selected.map(id => this.getContentNode(id));
+        const orderedNodes = selectedNodes.sort(this.compareNodeTitles);
```

Comment:
So this looks like it will sort the nodes alphabetically, which is appropriate. Although, there's an important distinction regarding how those are passed to `moveContentNodes`. This distinction is what led me to mention that it should be reversed in my directions on the issue.

For example, imagine a list of content nodes with fake titles listed below:
- Xylophone 
- Cat
- Animal
- Bat
- Zebra

Lets say the user selects the middle 3 for sorting:
- Cat
- Animal
- Bat
which means the expected sort would be:
- Animal
- Bat
- Cat

In this case, 'node X' is 'Xylophone'.

If your procedure here loops through them in that order, this is what will occur:
1. First loop moves 'Animal' after 'Xylophone' => Xylophone, Animal, Cat, Bat, Zebra
2. Second loop moves 'Bat' after 'Xylophone' => Xylophone, Bat, Animal, Cat, Zebra
3. Third loop moves 'Cat' after 'Xylophone' => Xylophone, Cat, Bat, Animal, Zebra

As you can see, the eventual order of the list becomes reversed. The behavior is the same regardless of using 'RIGHT' or 'FIRST_CHILD' positioning. So reversing the array after sorting should produce the proper alphabetical order in the end.